cmplibrary/src/main/java/com/sourcepoint/cmplibrary/SpConsentLibImpl.kt

```diff
@@ -208,6 +209,19 @@ internal class SpConsentLibImpl(
         }
     }
 
+    override fun customConsentGDPR(vendors: Array<String>, categories: Array<String>, legIntCategories: Array<String>, successCallback: CustomConsentClient) {
+        customConsentGDPR(
+            vendors = vendors.toList(),
+            categories = categories.toList(),
+            legIntCategories = legIntCategories.toList(),
+            success = {
+                it?.let { spc ->
+                    successCallback.transferCustomConsentToUnity(spc.toJsonObject().toString())
+                } ?: run { spClient.onError(RuntimeException("SPClient is null!")) }
```

Comment:
a more verbose message would be better, something like "An error occurred during the custom consent request"

Reply:
Fixed